gtsam/base/GenericValue.h

```diff
@@ -84,7 +84,7 @@ class GenericValue: public Value {
 
   /// Virtual print function, uses traits
   virtual void print(const std::string& str) const {
-    std::cout << "(" << demangle(typeid(T).name()) << ") ";
+    std::cout << "(" << demangle(typeid(T).name()) << ") " << std::endl;
```

Comment:
"\n" is preferred to std::endl since it might be faster. endl implies a stream sync, AFAIK.

This applies to everywhere in this PR...